frontend/src/components/payment/transactions/TransactionHistory.tsx

```diff
@@ -65,10 +93,44 @@ const TransactionHistory: FunctionComponent = () => {
       {error ? (
         <p>{t(error)}</p>
       ) : (
-        <TransactionList transactions={transactions} />
+        <>
+          <TransactionList
+            transactions={transactions.slice(
+              page * perPage,
+              page * perPage + perPage
+            )}
+          />
+          <div className={styles.controlPage}>
+            <Button
+              type='secondary'
+              onClick={() => setPage(0)}
+              disabled={page === 0}
+            >
+              {t('first-page')}
```

Comment:
Do we really need first page? Wouldn't expect that to be used often and throws of the balance of the buttons?

Reply:
Yeah let's rip it out, I was on the fence too